code/modules/shuttle/ferry.dm

```diff
@@ -1,5 +1,6 @@
 /obj/machinery/computer/shuttle/ferry
 	name = "transport ferry console"
+	desc = "A console that controls a transport ferry."

```

Comment:
I pondered the various perspectives it could be seen from, and thought this was for the best (also there is one OUTSIDE the ship too)